### ISSUE TITLE
Error when loading schema from graphql.config.yaml - schema.getQueryType is not a function

### ISSUE THREAD

**Issue by mogelbrod** [B2o5T/graphql-eslint#996]
**Describe the bug**
Attempting to run graphql linting relying on graphql.config.js currently appears to fail with the error:
```
$ DEBUG=graphql-eslint:* eslint empty.graphql
  graphql-eslint:parser cwd '.../repo' +0ms
  graphql-eslint:graphql-config options.skipGraphQLConfig: undefined +0ms
  graphql-eslint:graphql-config Graphql-config path '.../repo/graphql.config.yaml' +0ms
  graphql-eslint:schema Loading schema from 'https://countries.trevorblades.com/' +0ms
TypeError: schema.getQueryType is not a function
    at getRootTypeMap (.../repo/node_modules/@graphql-tools/wrap/node_modules/@graphql-tools/utils/index.js:713:30)
    at Object.memoized [as getRootTypeMap] (.../repo/node_modules/@graphql-tools/wrap/node_modules/@graphql-tools/utils/index.js:519:30)
    at generateProxyingResolvers (.../repo/node_modules/@graphql-tools/wrap/index.js:15:31)
    at Object.wrapSchema (.../repo/node_modules/@graphql-tools/wrap/index.js:80:31)
    at UrlLoader.loadSync (.../repo/node_modules/@graphql-tools/url-loader/index.js:801:30)
    at loadFileSync (.../repo/node_modules/@graphql-tools/load/index.js:115:46)
    at .../repo/node_modules/@graphql-tools/load/index.js:399:25
    at Object.runAll (.../repo/node_modules/@graphql-tools/load/index.js:245:17)
    at collectSourcesSync (.../repo/node_modules/@graphql-tools/load/index.js:291:11)
    at loadTypedefsSync (.../repo/node_modules/@graphql-tools/load/index.js:540:21)
error [graphql-eslint] Error while loading schema
 TypeError: schema.getQueryType is not a function

Oops! Something went wrong! :(

ESLint: 8.11.0

Error: Error while loading rule '@graphql-eslint/require-id-when-available': Rule 'require-id-when-available' requires 'parserOptions.schema' to be set and schema to be loaded. See http://bit.ly/graphql-eslint-schema for more info
Occurred while linting .../repo/empty.graphql
    at requireGraphQLSchemaFromContext (.../repo/node_modules/@graphql-eslint/eslint-plugin/index.js:38:15)
    at Object.create (.../repo/node_modules/@graphql-eslint/eslint-plugin/index.js:3066:24)
    at createRuleListeners (.../repo/node_modules/eslint/lib/linter/linter.js:922:21)
    at .../repo/node_modules/eslint/lib/linter/linter.js:1104:31
    at Array.forEach (<anonymous>)
    at runRules (.../repo/node_modules/eslint/lib/linter/linter.js:1041:34)
    at Linter._verifyWithoutProcessors (.../repo/node_modules/eslint/lib/linter/linter.js:1389:31)
    at Linter._verifyWithConfigArray (.../repo/node_modules/eslint/lib/linter/linter.js:1729:21)
    at Linter.verify (.../repo/node_modules/eslint/lib/linter/linter.js:1471:65)
    at Linter.verifyAndFix (.../repo/node_modules/eslint/lib/linter/linter.js:1953:29)
```

**To Reproduce**
```sh
mkdir repro
cd repro
npm i @graphql-eslint/eslint-plugin@3.9.1 eslint@8.11.0 graphql@16.3.0
cat << EOF > .eslintrc.js
module.exports = {
  root: true,
  overrides: [
    {
      files: ['*.graphql'],
      plugins: ['@graphql-eslint'],
      parser: '@graphql-eslint/eslint-plugin',
      parserOptions: { operations: './*.graphql' },
      extends: 'plugin:@graphql-eslint/operations-recommended',
      rules: { '@graphql-eslint/require-id-when-available': 2 }
    }
  ],
}
EOF
echo 'schema: "https://countries.trevorblades.com/"' > graphql.config.yaml
echo 'query DeleteMe { me { __typename } }' > empty.graphql
DEBUG=graphql-eslint:* eslint empty.graphql
```

**Expected behavior**
Linting errors related to the actual query

**Environment:**

- Mac OS Monterey (M1)
- @graphql-eslint/eslint-plugin@3.9.1
- eslint@8.11.0
- graphql@16.3.0
- NodeJS v17.5.0

**Comment by B2o5T** [comment]
Hi @mogelbrod, thank you for your report and reproduction!
I was able to reproduce, seems like the error goes from `graphql-tools` and not from `graphql-eslint`.
I leave this issue open until we fix `graphql-tools`.

Will back soon 🦸‍♂️

**Comment by mogelbrod** [comment]
@B2o5T Got it! Do you want me to file an issue on https://github.com/ardatan/graphql-tools/issues?

**Comment by B2o5T** [comment]
@mogelbrod no need, I already told 👌 fix with test https://github.com/ardatan/graphql-tools/pull/4334 was merged, so I’ll release a fix for graphql-eslint also soon.

**Comment by B2o5T** [comment]
fixed in [@graphql-eslint/eslint-plugin@3.10.0](https://github.com/B2o5T/graphql-eslint/releases/tag/%40graphql-eslint%2Feslint-plugin%403.10.0)
